stdlib/public/core/StringBridge.swift

```diff
@@ -507,6 +562,31 @@ internal func _SwiftCreateBridgedString(
   return Unmanaged<AnyObject>.passRetained(str._bridgeToObjectiveCImpl())
 }
 
+@available(macOS, introduced: 9999, deprecated)
+@available(iOS, introduced: 9999, deprecated)
+@available(watchOS, introduced: 9999, deprecated)
+@available(tvOS, introduced: 9999, deprecated)
+@available(*, deprecated)
+@_cdecl("_SwiftCreateBridgedMutableString")
+@usableFromInline
+internal func _SwiftCreateBridgedMutableString(
+  bytes: UnsafePointer<UInt8>,
+  length: Int,
+  encoding: _swift_shims_CFStringEncoding
+) -> Unmanaged<AnyObject> {
+  let bufPtr = UnsafeBufferPointer(start: bytes, count: length)
+  let str:String
+  switch encoding {
+  case kCFStringEncodingUTF8:
+    str = String(decoding: bufPtr, as: Unicode.UTF8.self)
+  case kCFStringEncodingASCII:
+    str = String(decoding: bufPtr, as: Unicode.ASCII.self)
+  default:
+    fatalError("Unsupported encoding in shim")
+  }
+  return Unmanaged<AnyObject>.passRetained(_SwiftNSMutableString(str))
+}
+
```

Comment:
This is deprecated already?

Reply:
This was the best Jordan and I could come up with for "won't get optimized out, will emit a symbol, isn't callable from Swift" :/